specification/resources/resource-manager/readme.md

```diff
@@ -80,6 +80,14 @@ These settings apply only when `--tag=package-policy-2020-09` is specified on th
 ``` yaml $(tag) == 'package-policy-2020-09'
 input-file:
 - Microsoft.Authorization/stable/2020-09-01/dataPolicyManifests.json
+- Microsoft.Authorization/stable/2020-09-01/policyAssignments.json
+- Microsoft.Authorization/stable/2020-09-01/policyDefinitions.json
+- Microsoft.Authorization/stable/2020-09-01/policySetDefinitions.json
+- Microsoft.Authorization/preview/2020-07-01-preview/policyExemptions.json
```

Comment:
I have a question about this preview swagger: will we have a stable version of this very soon?
If not, could we create a new tag that contains only the stable swaggers?

Reply:
It'll be a bit before its non-preview as it has dependency on an upcoming billing API which is not present yet.  I don't have  a problem separating the tag but when I generate SDKs I need to include all these specs currently in this tag and I was under the impression SDK generation is only done on a single tag.  Is that an incorrect assumption?

Reply:
That is true. But one side-effect is that (only speaking from azure-sdk-for-go) if one of the swagger files is preview, the whole package generated from that tag will be marked as preview, and some customers have concerns on using a preview package because preview package could receive breaking changes any time. Therefore since we already have this preview swagger included in a previous tag `package-2020-03-01`, could we first omit this preview package from this tag? 
And also it is easy to add a swagger file to an existing tag, but it is nearly impossible to remove a swagger file from an existing tag, which will cause a very big breaking change.

Reply:
@pilor , could you reply the above comment? Thanks.

Reply:
Sorry, was out due to US holidays.  Makes sense.  Will remove from the new tag

Reply:
@ArcturusZhang , could you confirm if the change looks good? Thanks.